includes/classes/order.php

```diff
@@ -687,6 +691,8 @@ public function do_calculate_taxes($taxRates, $index, $taxCountryId, $taxZoneId)
               $this->info['tax_groups'][$taxDescription] = $taxAdd;
           }
       }
+      
+      $this->notify('NOTIFY_ORDER_CART_TAX_CALCULATIONS_END', array('index' => $index, 'tax_add' => $tax_add, 'shown_price' => $shown_price, 'taxRates' => $taxRates));
```

Comment:
Are you passing this as an array because you primarily intend to use the observer for logging/reporting/recording what was done? ie: just watching? 

(Just curious.)